easybuild/tools/repository/svnrepo.py

```diff
@@ -88,8 +88,8 @@ def setup_repo(self):
         try:
             pysvn.ClientError  # IGNORE:E0611 pysvn fails to recognize ClientError is available
         except NameError, err:
-            self.log.exception("pysvn not available (%s). Make sure it is installed " % err +
-                               "properly. Run 'python -c \"import pysvn\"' to test.")
+            self.log.exception("pysvn not available (%s). Make sure it is installed properly." 
+                                + " Run 'python -c \"import pysvn\"' to test.", err)
```

Comment:
log.error is not a default logging function, so there the `, err` breaks things. These do not break for log.exception.
log.exception was designed to be used in an exception handling case, so it fits perfectly here. I don't like to use log.error, it has weird behaviour.